Services/Images/ImageService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Images;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System.IO;
+
+namespace Andculture.Sitefinity.Testing.Services.Images
+{
+    public class ImageService : ODataService<Image>
+    {
+        public ImageService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/images";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.